spyder/plugins/configdialog.py

```diff
@@ -398,7 +401,13 @@ def save_to_conf(self):
         for radiobutton, (option, _default) in list(self.radiobuttons.items()):
             self.set_option(option, radiobutton.isChecked())
         for lineedit, (option, _default) in list(self.lineedits.items()):
-            self.set_option(option, to_text_string(lineedit.text()))
+            option_value = lineedit.text()
+            old_value = self.get_option(option, _default)
+            if isinstance(old_value, list):
+                option_value = [int(e) for e in option_value.split(',')]

```

Comment:
@rlaverde, since this is a textbox, and we cannot safely assume the type of content we need to either NOT make it a list of ints, and handle that on the widget (check if int if not turn into an int), otherwise we will need to define a subtype of textbox that is explicit about the type... or add some extra options so we can safely define if it is a list and what type of items

https://github.com/spyder-ide/spyder/blob/master/spyder/plugins/configdialog.py#L467

@ccordoba12 ?